core/GlobalState.cc

```diff
@@ -1743,21 +1934,29 @@ unique_ptr<GlobalStateHash> GlobalState::hash() const {
     u4 hierarchyHash = 0;
     UnorderedMap<NameHash, u4> methodHashes;
     int counter = 0;
-    for (const auto &sym : this->symbols) {
-        if (!sym.ignoreInHashing(*this)) {
-            if (sym.isMethod()) {
-                auto &target = methodHashes[NameHash(*this, sym.name.data(*this))];
-                target = mix(target, sym.hash(*this));
-                hierarchyHash = mix(hierarchyHash, sym.methodShapeHash(*this));
-            } else {
```

Comment:
This could have hashed other symbol types than method/class before. Should we keep that capability?

Reply:
It didn't, though. `ignoreInHashing` filtered out non-methods and classes:
https://github.com/sorbet/sorbet/blob/b8b35b39/core/Symbols.cc#L1200

I removed the call because I can iterate over the symbols we want directly.

Reply:
Whoops, misread this yesterday. I need to re-add this in.

Reply:
Fixed.